posts/static/makepost/posts.js

```diff
@@ -106,24 +106,40 @@ function makePost(imageIDs = undefined) {
 
 function postImage(fileContent) {
     return new Promise((resolve, reject) => {
-
         let fileType = fileContent.split(':')[1].split(';')[0];
         fileType += ';base64';
-        let formData = new FormData();
-        let visbility = document.getElementById('visibility').value;
+        let visibility = document.getElementById('visibility').value;
         let csrf = document.getElementsByName('csrfmiddlewaretoken')[0].value;
-        formData.append('title', 'Image"');
-        formData.append('content', fileContent);
-        formData.append('description', 'Image');
-        formData.append('unlisted', 'true');
-        formData.append('contentType', fileType);
-        formData.append('visbility', visbility);
-        formData.append('csrfmiddlewaretoken', csrf);
+        let title = 'Image';
```

Comment:
Is there a reason that you changed from using formdata?
https://developer.mozilla.org/en-US/docs/Web/API/FormData/append

Reply:
I thought the name would need a [] suffix in order to append to the same name multiple times